src/api.rs

```diff
@@ -186,6 +173,8 @@ pub struct TransformBlock {
     random_transform_public_key: PublicKey,
     /// encrypted temp key value. Used to go from the transformed value to the encrypted value
     encrypted_random_transform_temp_key: EncryptedTempKey,
+    #[derivative(PartialEq = "ignore")]
+    #[derivative(Hash = "ignore")]
```

Comment:
These could be combined as:
```suggestion
    #[derivative(PartialEq = "ignore", Hash = "ignore")]
```